spec/classes/redis_spec.rb

```diff
@@ -43,8 +43,8 @@
 
         it do
           is_expected.to contain_service(service_name).with(
-            'ensure'     => 'running',
-            'enable'     => 'true',
+            'ensure' => 'running',
+            'enable' => 'true',
```

Comment:
Generally it makes it easier to review if you separate out the visual changes to a separate commit.

Reply:
Removed from the scope of this changes